### PR TITLE
[fix] CI notify-community-server

### PR DESCRIPTION
**Pull request by phoinixgrr**
#### Summary

Aligning the action with the breaking changes introduced under https://github.com/mattermost/action-mattermost-notify/pull/13

Context: https://community.mattermost.com/core/pl/yb9ei7obg7ntjr483dbo1j8yoo

#### Ticket Link

 https://mattermost.atlassian.net/browse/CLD-8100

### CODE REVIEW

**Review by toninis — APPROVED** [review]

File: .github/workflows/notify-mm-blog.yml [hunk 1]
```diff
@@ -19,10 +19,10 @@ jobs:
           .github/workflows/notify-mm-blog.sh
       - name: Notify channel 1
         uses: mattermost/action-mattermost-notify@master
```
Comment (hanzei) [inline .github/workflows/notify-mm-blog.yml:21]: Should we pin the version to avoid breaking changes?

Comment (phoinixgrr) [reply .github/workflows/notify-mm-blog.yml:21]: Let's do that, rebasing and pushing the recommendation

Comment (phoinixgrr) [reply .github/workflows/notify-mm-blog.yml]: @hanzei done https://github.com/mattermost/mattermost-marketplace/compare/3bfdb9c9353eb89a9790b02cc57476da839b44f8..84010f4ad0fd719a3ecb97af082fda213d980869

**Review by toninis — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Comment by phoinixgrr** [comment]
@hanzei Feel free to chery-pick this to all affected branches. `production`

**Comment by hanzei** [comment]
Hm, this doesn't seem to have done the trick. https://github.com/mattermost/mattermost-marketplace/actions/runs/10353510589 reports an error: `You need to provide TEXT or PAYLOAD input`

@phoinixgrr Can you please take a look?

**Comment by phoinixgrr** [comment]
> Hm, this doesn't seem to have done the trick. https://github.com/mattermost/mattermost-marketplace/actions/runs/10353510589 reports an error: `You need to provide TEXT or PAYLOAD input`
> 
> @phoinixgrr Can you please take a look?

@hanzei 
Ah yes. It needs a payload. What to actually send to the channels 
https://github.com/mattermost/action-mattermost-notify?tab=readme-ov-file#inputs

What would be that?

**Comment by phoinixgrr** [comment]
Hmm I see. 

There is a bash script that needs to be alligned. https://github.com/mattermost/mattermost-marketplace/blob/master/.github/workflows/notify-mm-blog.sh

**Pull request merged**